src/migrateCalculatedMeasuresToActivePivot/removeCalculatedMemberDefinitionFromMDXAndGetCubeName.ts

```diff
@@ -0,0 +1,48 @@
+import { DataModel, CubeName, getCube } from "@activeviam/data-model";
+import {
+  MdxString,
+  MdxSelect,
+  getCalculatedMeasures,
+  getCubeName,
+  removeCalculatedMemberDefinition,
+  stringify,
+} from "@activeviam/mdx";
+import { parse } from "@activeviam/activeui-sdk";
+import _intersection from "lodash/intersection";
+
+export const removeCalculatedMemberDefinitionFromMDXAndGetCubeName = (
+  mdx: MdxString,
+  namesOfCalculatedMeasurestoMigrate: string[],
+  dataModel: DataModel,
+): {
+  calculatedMeasuresWithCubeNames: { [measureName: string]: CubeName };
+  stringifiedUpdatedMdx: MdxString;
+} => {
+  const calculatedMeasuresWithCubeNames: { [measureName: string]: CubeName } =
+    {};
+  const parsedMdx: MdxSelect = parse(mdx);
+
+  const calculatedMeasuresKeysUsedByWidgetToBeMigrated = _intersection(
+    // The keys are the names of the calculated measures, a widget may have more than one calculated measure.
```

Comment:
I don't think we need to specify that a widget may have several calculated measures in its query.
It's the MDX spec, and the function name `getCalculatedMeasures` is plural anyway.
About the variable name, I would align it on the function argument.
```suggestion
  const namesOfCalculatedMeasuresToMigrateInWidget = _intersection(
```